### PR TITLE
lint: fix untyped-defs on /tests/unittest/cmd

### PR DESCRIPTION
**Pull request by wideawakening**
related to https://github.com/canonical/cloud-init/issues/5445

effort has been timeboxed, as part of the onboarding.

- [x] tests.unittests.cmd.devel.test_hotplug_hook
- [x] tests.unittests.cmd.devel.test_render
- [x] tests.unittests.cmd.test_clean
- [x] tests.unittests.cmd.test_query
- [x] tests.unittests.cmd.test_status

## Proposed Commit Message

```
lint: fix untyped-defs on /tests/unittest/cmd
```

## Merge type

- [x] Squash merge using "Proposed Commit Message"
- [ ] Rebase and merge unique commits. Requires commit messages per-commit each referencing the pull request number (#<PR_NUM>)

### CODE REVIEW

**Comment by holmanb** [comment]
Hi @wideawakening o/

Thanks for contributing to cloud-init. This looks like a great first addition to cloud-init. Can you please sign [the CLA](https://ubuntu.com/legal/contributors) so we can accept this contribution?

**Comment by wideawakening** [comment]
hey, sure.
should have been accepted already

File: tests/unittests/cmd/devel/test_hotplug_hook.py [hunk 1]
```diff
@@ -19,6 +20,14 @@
 FAKE_MAC = "11:22:33:44:55:66"
 
 
+# Define the NamedTuple class
```
Comment (aciba90) [inline tests/unittests/cmd/devel/test_hotplug_hook.py:23]: Nit: This comment is redundant, IMO. Could you please remove it?

**Review by aciba90 — APPROVED** [review]
LGTM. Many thanks for making cloud-init better, welcome!

**Pull request merged**